MekHQ/src/mekhq/campaign/finances/Money.java

```diff
@@ -42,6 +41,10 @@ private Money(BigMoney money) {
         this.wrapped = money;
     }
 
+    public BigMoney getWrapped() {
```

Comment:
Does this need to be public?

Reply:
It does not, just a normal access level for a getter

Reply:
Okay, when @VicenteCartas worked on this BigMoney was meant as an implementation detail not to be exposed.